fabbwled-backend/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.15.2'
```

Comment:
can the version be dropped because it is already defined in the spring-bom? /notSure...

Reply:
I don't know either, but it seems to work so having the version there is probably fine. I don't know _nearly_ enough about maven and gradle for this